scripts/commands/getstats.lua

```diff
@@ -32,19 +32,24 @@ function onTrigger(player)
     player:PrintToPlayer(string.format("HP: %i/%i  MP: %i/%i (current/max) ",
         target:getHP(), target:getMaxHP(), target:getMP(), target:getMaxMP()), xi.msg.channel.SYSTEM_3)
 
-    player:PrintToPlayer(string.format("Food Accuracy%% bonus: %i ", target:getMod(xi.mod.FOOD_ACCP)), xi.msg.channel.SYSTEM_3)
-    player:PrintToPlayer(string.format("Accuracy bonus: %i ", target:getMod(xi.mod.ACC)), xi.msg.channel.SYSTEM_3)
-    player:PrintToPlayer(string.format("EVA Bonus: %i ", target:getMod(xi.mod.EVA)), xi.msg.channel.SYSTEM_3)
     player:PrintToPlayer(string.format("Total STR: %i ", target:getStat(xi.mod.STR)), xi.msg.channel.SYSTEM_3)
     player:PrintToPlayer(string.format("Total DEX: %i ", target:getStat(xi.mod.DEX)), xi.msg.channel.SYSTEM_3)
     player:PrintToPlayer(string.format("Total VIT: %i ", target:getStat(xi.mod.VIT)), xi.msg.channel.SYSTEM_3)
     player:PrintToPlayer(string.format("Total AGI: %i ", target:getStat(xi.mod.AGI)), xi.msg.channel.SYSTEM_3)
     player:PrintToPlayer(string.format("Total MND: %i ", target:getStat(xi.mod.MND)), xi.msg.channel.SYSTEM_3)
     player:PrintToPlayer(string.format("Total INT: %i ", target:getStat(xi.mod.INT)), xi.msg.channel.SYSTEM_3)
     player:PrintToPlayer(string.format("Total CHR: %i ", target:getStat(xi.mod.CHR)), xi.msg.channel.SYSTEM_3)
-    player:PrintToPlayer(string.format("Attack bonus: %i ", target:getMod(xi.mod.ATT)), xi.msg.channel.SYSTEM_3)
+    player:PrintToPlayer(string.format("Food Accuracy%% bonus: %i ", target:getMod(xi.mod.FOOD_ACCP)), xi.msg.channel.SYSTEM_3)
+    player:PrintToPlayer(string.format("Accuracy Base: %i ", target:getMod(xi.mod.ACC)), xi.msg.channel.SYSTEM_3)
+    player:PrintToPlayer(string.format("Total Accuracy: %i ", target:getStat(xi.mod.ACC)), xi.msg.channel.SYSTEM_3)
+    player:PrintToPlayer(string.format("RAccuracy Base: %i ", target:getMod(xi.mod.RACC)), xi.msg.channel.SYSTEM_3)
+    player:PrintToPlayer(string.format("Total RAccuracy: %i ", target:getStat(xi.mod.RACC)), xi.msg.channel.SYSTEM_3)
+    player:PrintToPlayer(string.format("EVA Base: %i ", target:getMod(xi.mod.EVA)), xi.msg.channel.SYSTEM_3)
```

Comment:
just a note for future if you'd like to make any additions: I had originally planed to have a another parameter on the command and split up into multiple optional pages, as it gets kinda spammy. So further moderation to this are quit welcome. 👍